topsort-endpoints.yml

```diff
@@ -9,16 +9,16 @@ info:
   version: 1.0.0
 
 servers:
-- url: https://demo.api.topsort.com/v1
-  description: Demo server (uses test data)
-- url: https://api.topsort.com/v1
-  description: Production server (uses live data)
+  - url: https://demo.api.topsort.com/v1
+    description: Demo server (uses test data)
+  - url: https://api.topsort.com/v1
```

Comment:
The prod hostnames are <marketplace-slug>.ai.topsort.ai

Reply:
Should I remove the `api.topsort.com` server?
Should we rename the demo one to use the new domain?

Reply:
Yes, when we give people (in the future) sandbox, staging, or demo marketplaces we should give the corresponding <marketplace>.api.topsort.ai URLs so that their (and our) dev environments are more similar to production. ATM the marketplace-specific urls don't work for manager.